app/src/main/java/org/liberty/android/fantastischmemo/ui/QuizActivity.java

```diff
@@ -153,14 +154,28 @@ public void onCreate(Bundle savedInstanceState) {
     public void startTimer() {
         this.countDownTimer = new CountDownTimer(timeLeftInMilliseconds, 1000) {
 
+            MediaPlayer mediaPlayer = MediaPlayer.create(getApplicationContext(), R.raw.countdown);
+
             @Override
             public void onTick(long millisUntilFinished) {
                 timeLeftInMilliseconds = millisUntilFinished;
+
+                if(!mediaPlayer.isPlaying()) {
+                    if (timeLeftInMilliseconds <= 8000) {
+                        mediaPlayer.start();
+                    }
+                }
+
+                if(timeLeftInMilliseconds <= 11000){
```

Comment:
need space after if and after closing parenthesis 
if () {

Reply:
Fixed!